apps/sms/services/js/moz_mobile_message/messaging_service.js

```diff
@@ -0,0 +1,193 @@
+/*global BridgeServiceMixin, Settings */
+'use strict';
+
+if (!('BridgeServiceMixin' in self)) {
+  importScripts('/services/js/bridge_service_mixin.js');
```

Comment:
wondering if it would be better to use ```self.importScripts``` - will be a bit easier to mock this function in tests :)